app/react/components/AdvocateLogin.jsx

```diff
@@ -39,6 +40,14 @@ class AdvocateLogin extends Component {
                     Submit
                   </button>
                 </div>
+                {
+                  this.props.loginError &&
+                    (
+                      <div className="login-error">
+                        <p>Login Failed</p>
```

Comment:
Can we say something slightly...kinder, for lack of a better word? Like, "Something went wrong during your login attempt. Please try again."?